src/KM_Defaults.pas

```diff
@@ -674,6 +674,11 @@ interface
   icRed    = $FF0707FF;
 
   icSteelBlue = $FFA56D53; // Selection color
+  icRoyalYellow = $FF4AC7FF;
```

Comment:
We decided to do some refactoring on skype, smth like
clRed = 
clPingHigh = clRed
clFpsLow = clRed

Here, similar: